ai-assistant/README.md

```diff
@@ -0,0 +1,27 @@
+# AI Assistant editor
+
+Add powerful AI content creation tools into CKEditor. Use pre-made prompts or custom queries to have AI write, rephrase, translate or summarize your text.
+
+This demo showcases most of the CKEditor features along with some premium addons: Export to PDF/Word, Import from Word.
```

Comment:
It's a nitpick, but maybe we could rephrase this sentence a bit:
> This demo showcases most CKEditor features alongside some premium add-ons: Export to PDF/Word and Import from Word.

Reply:
Changed.